src/logic/test/calculate.test.js

```diff
@@ -0,0 +1,9 @@
+import calculate from '../calculate';
+
```

Comment:
- [ ] Please cover the remaining functionalities of this script like the +/- and the dot functionality to comply with the project requirements.